docs/source/getting-started/statistical-modeling/synthetic-data.rst

```diff
@@ -0,0 +1,148 @@
+.. _synthetic-data:
+
+Synthetic Data
+==============
+
+Synthetic data mechanisms in OpenDP attempt to model the relationships between columns in your data via a contingency table,
+and then generate synthetic data with the same relationships.
+
+OpenDP uses `Private-PGM <https://private-pgm.readthedocs.io/en/latest/introduction.html>`_ 
+to build a contingency table that is consistent with a set of marginal queries via a process called model-based inference (``mbi``).
+This functionality is not enabled by default.
+
+.. tab-set::
+
+    .. tab-item:: Python
+        :sync: python
+
+        Add ``[mbi]`` to the package installation to enable model-based-inference functionality.
+
+        .. prompt:: bash
+
+            pip install 'opendp[mbi]'
+
+        This installs the specific version of the Private-PGM package that is compatible with the OpenDP Library.
+
+    .. tab-item:: R
+        :sync: r
+
+        ``mbi`` (Private-PGM) is only available in Python. 
+
+    .. tab-item:: Rust
+        :sync: rust
+
+        ``mbi`` (Private-PGM) is only available in Python. 
+
+Let's get started by setting up the context for the Labor Force dataset.
+
+.. code:: pycon
+
+    >>> import opendp.prelude as dp
+    >>> import polars as pl
+
+    >>> dp.enable_features("contrib")
+
+    >>> context = dp.Context.compositor(
+    ...     data=pl.scan_csv(dp.examples.get_france_lfs_path(), ignore_errors=True),
+    ...     privacy_unit=dp.unit_of(contributions=36),
+    ...     privacy_loss=dp.loss_of(rho=0.2, delta=2e-7),
+    ... )
+
+
+The :ref:`contingency table <contingency-tables>` documentation is recommended reading 
+and gives an introduction to the problem,
+but differs in that the query workload is fixed (using the :py:class:`opendp.extras.mbi.Fixed` algorithm).
+All algorithms for differentially private contingency table estimation inherit from :py:class:`opendp.extras.mbi.Algorithm`:
+these being :py:class:`opendp.extras.mbi.Fixed`, :py:class:`opendp.extras.mbi.AIM`, :py:class:`opendp.extras.mbi.MST` and :py:class:`opendp.extras.mbi.Sequential`.
+
+The algorithms for synthetic data differ from the fixed algorithm
+in that they first estimate all unknown first-order marginals, not just the marginals with missing keys.
+This behavior can be controlled by switching between ``.oneway="all"`` and ``.oneway="unkeyed"``
+in the algorithm of choice.
```

Comment:
For getting started, I think we should just pick one, and leave the rest, and the comparison between the choices, to the API User Guide.